src/import-data/ImportData.ts

```diff
@@ -63,12 +63,20 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
   });
 
   const importPFB = async (importRequest: PFBImportRequest, workspace: WorkspaceInfo) => {
-    const { namespace, name } = workspace;
-    const { jobId } = await Ajax()
-      .Workspaces.workspace(namespace, name)
-      .importJob(importRequest.url.toString(), 'pfb', null);
-    asyncImportJobStore.update(Utils.append({ targetWorkspace: { namespace, name }, jobId }));
-    notifyDataImportProgress(jobId);
+    if (workspace.cloudPlatform === 'Azure') {
+      const wdsUrl = await Ajax().Apps.listAppsV2(workspace.workspaceId).then(resolveWdsUrl);
+      await Ajax().WorkspaceData.startImportJob(wdsUrl, workspace.workspaceId, {
+        url: importRequest.url.toString(),
+        type: 'PFB',
```

Comment:
the divergence of upper case PFB vs lower case in the else jumped out at me. likely harder to align then the benefit we get from that, but could be error prone in future

Reply:
Yeah, it's the difference between the "type" of the import request object and the WDS type parameter.

They don't really align since the import request type has several more cases than the WDS type parameter.

https://github.com/DataBiosphere/terra-ui/blob/dev/src/import-data/import-types.ts

But yeah, generally, we're _wildly_ inconsistent about capitalization of enums/constants. Cloud provider is even more fun (Azure/Gcp for Workspaces, AZURE/GCP for Leo, and gcp/azure for the Catalog).